n2y/blocks.py

```diff
@@ -2,11 +2,11 @@
 import logging
 from urllib.parse import urljoin
 
-from pandoc.types import (
+from pandoc.types import (  # noqa: F401
```

Comment:
Why is F401 necessary here?

Reply:
the `Space` ast class is unused but the test suite throws this error if it's not imported:
```
n2y/blocks.py:423: in to_pandoc
    plain_children = eval(plain_children_string)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   NameError: name 'Space' is not defined

<string>:1: NameError
```

Reply:
Got rid of the eval statement so it's now unnecessary

Reply:
@sHermanGriffiths I'm not sure if you've pushed up your most recent code. I still see the "nova: F401" in the diff.